### DISCUSSION TITLE
Form View Bug [Dec 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4711] [Discord Threads]
> 🙋‍♂️ **Shard**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### please create an issue in Github with details.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1044166638024527952)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
